common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala

```diff
@@ -4,10 +4,16 @@ package com.gu.mediaservice.lib.config
 import scalaz._
 import Scalaz._
 
+
+
+import com.gu.mediaservice.model.{StaffPhotographer, ContractPhotographer, Photographer}
+
 object PhotographersList {
   type Store = Map[String, String]
   type CreditBylineMap = Map[String, List[String]]
 
+  import MetadataConfig.{ staffPhotographers, contractedPhotographers }
```

Comment:
how scala-rific (?) is it to have `import` statements within objects vs at top of file?